src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/Microsoft.Extensions.Diagnostics.ResourceMonitoring.csproj

```diff
@@ -6,6 +6,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
+    <NoWarn Condition="'$(TargetFramework)' == 'net462'">$(NoWarn);LA0006</NoWarn>
```

Comment:
NIT: Can we add a comment above and explain why are we turning off that one and what is it for?

Reply:
Done.